packages/samples/sources/move-basics/references.move

```diff
@@ -37,7 +39,7 @@ public fun enter_metro(card: &mut Card) {
 // ANCHOR: move
 /// Recycle the metro pass card.
 public fun recycle(card: Card) {
-    assert!(card.uses == 0, ENoUses);
+    assert!(card.uses == 0, EHasUses);
```

Comment:
I don't think this change is necessary for this example. Any card can be recycled.

Reply:
Good call, let's merge this bud!